### PR TITLE
Adding help section

### PR DESCRIPTION
**Pull request by inderpreetsingh**
In order to solve issue #38  I ended up doing couple of stuff that I believe will help us in adding future markdown based  static content windows.

* I refactored `openChangeLogWindow` function into `openSecondaryWindow` so that it can open any window we want it to by just passing an argument.
* Then used `openSecondaryWindow` to open a help/guide window.
* Refactored `changelog.js`  into `markdownToHTML` so that it can convert any markdown file into html and load into respective id.
* Refactored `changlog.scss` into `markdown.scss` to allow it to style all the markdown based files
* Added content into newly created `HELP.md`
* Added some typographic rules and vertical rhythm into `markdown.scss`. Now both changelog window and help window look beautiful <3 <3 
For now I am loading images and videos from website to not bloat the app size, however let me know if we want to load them into app.

### CODE REVIEW

File: HELP.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+###  Video Tutorial
+
+<div class="videoWrapper"><iframe width="100%" height="400px" src="https://www.youtube.com/embed/ZDX8nPkDBSc" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen=""></iframe></div>
```
Comment (navdeepsinghkhalsa) [inline HELP.md]: Could you please use "lowercase-with-dashes" for CSS classes and id's?

File: www/js/controller.js [hunk 2]
```diff
@@ -168,10 +168,16 @@ const winMenu = [
         enabled: false,
       },
       ...updateMenu,
+      {
+        label: 'Guide...',
```
Comment (navdeepsinghkhalsa) [inline www/js/controller.js]: Could we please put this in a Help menu?

File: www/js/controller.js [hunk 3]
```diff
@@ -188,10 +194,16 @@ const macMenu = [
         role: 'about',
       },
       ...updateMenu,
+      {
+        label: 'Guide...',
```
Comment (navdeepsinghkhalsa) [inline www/js/controller.js]: Same as above - under a new Help top-level menu

File: www/src/scss/_markdown.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
+$container: 700px;
+$base-font-size: 16px;
+$ratio:1.5;
```
Comment (navdeepsinghkhalsa) [inline www/src/scss/_markdown.scss]: Could you please add a space after the colon?

File: www/src/scss/_markdown.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  margin: auto;
+  max-width: $container;
+  padding: $vertical-rhythm;
+
+  h2,
+  h3,
+  h4 {
+    color: #333;
+    line-height: 1;
+    margin-bottom: $vertical-rhythm/2;
```
Comment (navdeepsinghkhalsa) [inline www/src/scss/_markdown.scss]: Could you please use spaces between operators?

Comment (navdeepsinghkhalsa) [reply www/src/scss/_markdown.scss]: All other instances as well

File: www/help.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Changelog</title>
```
Comment (navdeepsinghkhalsa) [inline www/help.html]: Could you please change the title? It should be `SikhiToTheMax Help`

File: www/js/controller.js (hunk 2 above)
Comment (inderpreetsingh) [reply www/js/controller.js]: I think this is already done!

Comment (navdeepsinghkhalsa) [reply www/js/controller.js]: Yes, you're right. Sorry

File: www/js/controller.js (hunk 3 above)
Comment (navdeepsinghkhalsa) [reply www/js/controller.js]: Could this please be moved to match macOS menubar? With a top-level menu called Help

**Comment by navdeepsinghkhalsa** [comment]
@inderpreetsingh Just one change left for this one and we're good to merge!

**Comment by inderpreetsingh** [comment]
@navdeepsinghkhalsa veerji, I don't have mac so wasn't able to test. I hope it works fine!

**Pull request merged**